charts/policy-hub/README.md

```diff
@@ -6,12 +6,6 @@ For further information please refer to [Technical Documentation](../../docs/tec
 
 The referenced container images are for demonstration purposes only.
 
-## Prerequisites
```

Comment:
Prerequisites have been removed by regeneration / main has not yet been merged.

Reply:
thank for point that out, I added the section now to `charts/policy-hub/README.md.gotmpl` so I don't overwrite it anymore with `helm-docs` anymore.